### ISSUE TITLE
ghc installation is way too verbose

### ISSUE THREAD

**Issue by m-renaud** [haskell/ghcup#37]
When running `ghcup install` it writes everything to the console that its doing in the background (several hundred lines) which I don't need to see unless something goes wrong.

If we were to hide it by default and dump it into a log file (maybe `~/.ghcup/.log` or something like that) then only important information is surfaced to the user and they're less likely to miss something important (like needing to update their PATH).

**Comment by m-renaud** [comment]
Yeah that makes sense, I would hope that it succeeds more often then it fails though ;P It looks like [these two lines](https://github.com/haskell/ghcup/blob/master/ghcup#L1020-L1021) could redirect their stdout and stderr to a log file (unless `--verbose`), and then print where logs are written if the command fails.

**Comment by hasufell** [comment]
Yes, possible. That has to be done for the compile command as well then, which are even more verbose. Probably there we want to make the *build* verbose though, but write it to a log file... but we still may want stderr to be sent to terminal

**Comment by hvr** [comment]
@hasufell can't you record to the build-log *and* in case of errors dump the output (or parts of it) to stderr or alternatively point the user to the log file ala

     An error occured, please consult ....path-to-log... for details

?

**Comment by hasufell** [comment]
I'm unsure whether this complexity is really that useful. The relevant information for updating e.g. PATH is presented after the installation, so I don't see how a user would miss something right now.

**Comment by jtdaugherty** [comment]
I'm also in favor of improving this. In situations like this I've had good results with the approach described above, since all of the relevant output can be captured and preserved in case of an error.
